build_web_compilers/pubspec.yaml

```diff
@@ -32,3 +32,6 @@ dev_dependencies:
     path: test/fixtures/a
   b:
     path: test/fixtures/b
+
+dependency_overrides:
```

Comment:
These will prevent us from publishing - I would undo these once we confirm it works fine on travis

Reply:
t'was the plan 😄